.github/scripts/spellcheck.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2022 Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# SPDX-License-Identifier: curl
+#
+# Docs: https://github.com/UnicornGlobal/spellcheck-github-actions
+matrix:
+- name: Markdown
+  expect_match: false
+  apsell:
```

Comment:
`apsell`? Not `aspell`?

Reply:
I've seen that as well. I decided not to touch it since it works. The `apsell` comes from the example config over here: https://github.com/marketplace/actions/check-spelling-js-vue-html-markdown-text

... where you can also see that both `aspell` and `apsell` are used. I suspect apsell is a typo but that English is the default anyway.